### ISSUE TITLE
The included squish causes compile to fail

### ISSUE THREAD

**Issue by eGax** [linux-source-tools/gimp-plugin-vtf#1]
I'm trying to build this on Manjaro Linux XFCE 64bit. I'm seeing quite a few -Wdeprecated-declarations dealing with a gimp before the fail, but the compile only stops when it gets to the part dealing with libsquish. I see the same statement for each of the _ZTVN6squish***FitE's listed below. 

`/usr/bin/ld: libsquish/lib/libsquish.a(squish.cpp.o): relocation R_X86_64_32S against symbol _ZTVN6squish***FitE' can not be used when making a shared object; recompile with -fPIC`

`_ZTVN6squish15SingleColourFitE`
`_ZTVN6squish10ClusterFitE`
`_ZTVN6squish9ColourFitE`
`_ZTVN6squish8RangeFitE`
`_ZTVN6squish15SingleColourFitE`

Basically, the fix was to dump your included squish in favor of installing it from the Arch User Repository before compiling the gimp-plugin-vtf code. Thought this might be useful for the less code compile friendly folks out there. It seems to be working fine with a fairly current 2.9 dev build of GIMP.

**Comment by Rahix** [comment]
Thank you for noticing, I haven't touched this project in a while. I also added a [PKGBUILD](https://gist.github.com/Rahix/a297457945ab236551c9ffbe5f6b9052) so installation on arch-like systems is even easier 😃
